tls/s2n_config.h

```diff
@@ -105,3 +104,5 @@ extern int s2n_config_free_session_ticket_keys(struct s2n_config *config);
 
 extern void s2n_wipe_static_configs(void);
 int s2n_config_get_cert_type(struct s2n_config *config, s2n_cert_type *cert_type);
+extern struct s2n_cert_chain_and_key *s2n_fetch_single_default_cert(struct s2n_config *config);
+extern int s2n_get_num_of_default_certs(struct s2n_config *config);
```

Comment:
namic nits: `s2n_config_get_num_default_certs` and `s2n_config_get_single_default_cert` to match convention.